seam/__init__.py

```diff
@@ -14,3 +14,4 @@
     SeamActionAttemptTimeoutError,
 )
 from .seam_webhook import SeamWebhook
+from svix.webhooks import WebhookVerificationError
```

Comment:
@razor-x, does it make sense to export the WebhookVerificationError raised by the verify method for SDK consumers to catch? Or should we catch it ourselves and raise a Seam-labeled error?

Reply:
Ah, nice call. We should do this in https://github.com/seamapi/javascript-webhook too.

Can we alias it to `SeamWebhookVerificationError`? I'm not sure if we will be able to do this in all language, but it seems off to have `SeamWebhook` but then `WebhookVerificationError`.

Reply:
16beac1006b82f84836043d44c60837fe6242f16